guides/slack-guide.mdx

```diff
@@ -0,0 +1,150 @@
+    } catch (error) {
+        console.log("notifController error:", error);
+        res.status(500).json({ message: error.message })
+    }
+}
+```
+<Info>Notice how we're expecting 'chatMsg' in our payload. This is why we added it as a variable in the workflow created earlier.</Info>
+
+To make it modular, we’ll keep the trigger code in a separate function in a separate file, `novu.js`, in our case, which is as follows:
+```jsx
+import { Novu, ChatProviderIdEnum } from '@novu/node';
+
+export const chat = async (chatMsg) => {
+    const novu = new Novu(process.env.YOUR_NOVU_API_KEY_HERE);
+    await novu.subscribers.identify(process.env.SUB_ID, {
+        firstName: 'newSubForSlackChat',
+    });
+
+    await novu.trigger('slack', {
+        to: {
+            subscriberId: process.env.SUB_ID
+        },
+        payload: {
+            chatMsg: chatMsg
+        }
+    });
+}
+```
+In this code snippet above, we're first initializing a new instance of Novu, then using it to 'identify' or [create](https://docs.novu.co/subscribers/subscribers#create-a-subscriber) a [subscriber](https://docs.novu.co/subscribers/subscribers). 
+<Info>The 'identify' method tries to find a subscriber with the given info. If it can't find any such subscriber, it creates a new subscriber with the supplied info.</Info>
+After having created the subscriber, it runs the trigger code for the workflow we had created. You can find the trigger workflow by clicking on the 'Get Snippet' button on the workflow:
+<Frame caption="Click the 'Get Snippet' button to get trigger code"><img src="/images/guides/slack-guide/trigger.png" /> </Frame>
+<Frame caption="Paste this trigger code in your backend"><img src="/images/guides/slack-guide/trigger-code.png" /> </Frame>
+
+# Configure the Slack app
+
+After having created the Slack app and the backend, there are two ways to configure the Slack app. One is the Novu managed option and the other is the manually managed option. 
+Since the Novu managed is fairly straightforward, I'll demonstrate the manual method here. Follow along to set it up!
+
+1. Goto 'Incoming Webhooks' in your app settings and turn it on.
+<Frame caption="Turn on 'Incoming Webhooks' in your Slack app"><img src="/images/guides/slack-guide/on.png" /> </Frame>
+2. Click on the 'Add New Webhook to Workspace':
+<Frame caption="Click the 'Add New Webhook to Workspace' button"><img src="/images/guides/slack-guide/add.png" /> </Frame>
+3. Now, go ahead and select the channel in which you want to send notifications and click 'allow'.
+<Frame caption="Select the channel"><img src="/images/guides/slack-guide/install.png" /> </Frame>
+4. Then, copy the 'webhookUrl' from Slack.
+<Frame caption="Copy the webhookUrl from Slack"><img src="/images/guides/slack-guide/hook.png" /> </Frame>
+5. Now, add this code to the backend trigger code. In our case, we'll add it to the `Novu.js` file we'd created earlier so that the final file becomes:
```

Comment:
=> Now, add the code below to the backend trigger functionality.

=> we instead of we'd

What code  are they adding? That code should have a section so they can understand. And you need to explain why they need to add that code.